tests/acceptance/features/apiSharingNg/listPermissions.feature

```diff
@@ -560,3 +560,169 @@ Feature: List a sharing permissions
         }
       }
       """
+
+  @issues-8351
+  Scenario: user lists root permissions of a project space
+    Given using spaces DAV path
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "new-space" with the default quota using the Graph API
+    When user "Alice" lists the root permissions of drive "new-space" using the Graph API
```

Comment:
different API has been hit so whats the point of merging two scenario

Reply:
Yes, but the step can be confusing later on because we are doing the same thing but with different endpoints. We need better distinction for these steps. If we can implement an example table for endpoints or the other way would be great.

Reply:
if you want to reword, please suggest some but I don't think implementing two different api in one step is nice thing to do @saw-jan

Reply:
I suggest renaming the steps:
```diff
- When user "Alice" lists the permissions of space "new-space" using the Graph API
+ When user "Alice" lists the permissions of space "new-space" using permissions endpoint of Graph API
```
```diff
- When user "Alice" lists the root permissions of drive "new-space" using the Graph API
+ When user "Alice" lists the permissions of space "new-space" using root endpoint of Graph API
```

https://owncloud.dev/libre-graph-api/#/drives.permissions